packages/wallets/src/components/Base/WalletTextField/WalletTextField.scss

```diff
@@ -46,6 +46,7 @@
     }
 
     &__field {
+        min-width: 0;
```

Comment:
This fixes the "arrow icon got shifted to the right because input wouldn't shrink beyond a certain point" part of this

![image](https://github.com/binary-com/deriv-app/assets/119863957/ca967567-23ce-4f38-ac0f-8d1738e0123e)